src/implementations/twig/utilities/default/index.story.js

```diff
@@ -0,0 +1,74 @@
+import withCode from '@ecl/storybook-addon-code';
+
+export default {
+  title: 'Utilities/HTML tag styling',
+  decorators: [withCode],
+  parameters: {
```

Comment:
I think we should enable the html tags css in this story by default.

Reply:
do we have an example somewhere, where we activate a css for a specific story?

Reply:
Surely we have it in OEL. you have to override the cssresources in the parameters, basically..